HTML/1. intro.md

```diff
@@ -0,0 +1,33 @@
+# Intro
+
+![HTML](https://img.17qq.com/images/mhsgsnffsky.jpeg)
+
+HTML is a markup language, meaning that it is used to create webpages, so it's not like a normal programming language like Python
+
+___
+
+HTML is structured by tags, which make elements, here is an example of a tag - `<html>`
+
+And here is an example of an element `<html></html>`
+
+Each element has a opening tag and a closing tag, the opening tag has a `<` to begin and a `>` to end. The closing tag has a `<` to begin and a `/>` to end. Everything else between the opening and closing tags is either content or other nested elements
+
+There is an exception though, there are self closing tags, that don't have any content between them and they can be structered in two ways. With a `<` to begin and a `>` to end. And a `<` to begin and a `/>` to end.
+
+___
+
+Here is the neccesary code to start a webpage
+
+```html
+<!DOCTYPE html>
```

Comment:
You never introduced the "doctype" element, and many web developers have no clue what it does.

Reply:
The definition that was recently added is incorrect, you may want to research what it does yourself, look into on Wikipedia and MDN.